pkg/kubelet/kubelet.go

```diff
@@ -901,6 +901,23 @@ func (kl *Kubelet) initialNodeStatus() (*api.Node, error) {
 		if err != nil {
 			return nil, err
 		}
+
+		// If the cloud has zone information, label the node with the zone information
+		zones, ok := kl.cloud.Zones()
```

Comment:
@justinsb where we will use this information to have the scheduler spread pods in an RC across zones when a cluster is running in multiple-AZs.
+1
we have done it in version 0.12，and migrate it to 1.0. we tag zones information with label ,maybe can move it into nodeSpec,because kubelet can get zones in VM exposed by IaaS.